app/test_main.py

```diff
@@ -1,3 +1,53 @@
 from app.main import get_human_age
+import pytest
 
-# write your code here
+
+def test_get_human_age_zero() -> None:
+    assert get_human_age(0, 0) == [0, 0], \
+        "Expected [0, 0] for 0 cat and 0 dog age"
+
+
+def test_get_human_age_first_year() -> None:
+    assert get_human_age(14, 14) == [0, 0], \
+        "Expected [0, 0] for 14 cat and 14 dog age"
+
+
+def test_get_human_age_second_year() -> None:
+    assert get_human_age(23, 23) == [1, 1], \
```

Comment:
The expected output for the second year should not be [1, 1] if the function is supposed to convert pet years to human years. Please verify the logic and expected output.